RELEASES.md

```diff
@@ -1,3 +1,125 @@
+Version 1.55.0 (2021-09-09)
+============================
+
+Language
+--------
+- [You can create bindings inside of a `@` bindings pattern.][85305] E.g. the
+  following is now possible.
+  ```rust
+  let outer_option = Some(42);
+
+  match outer_option {
+      option @ Some(value) => {
+         // We can now use both `option` and `value`.
+         ...
+      }
+      _ => { ... }
+  }
+  ```
+- [You can now write open "from" ranges (`X..`), which will start at `X` and
+  will end at the maximum value of the integer.][83918]
+- [You can now access `union` fields in `const` contexts.][85769]
+- [You can now explicitly import the prelude of different editions
+  through `std::prelude`. (e.g. `use std::prelude::rust_2021::*;`)][86294]
+
+Compiler
+--------
+- [Added tier 3\* support for `powerpc-unknown-freebsd`.][87370]
+- [Added tier 3 support for `powerpc64le-unknown-freebsd`.][83572]
+
+\* Refer to Rust's [platform support page][platform-support-doc] for more
+   information on Rust's tiered platform support.
+
+Libraries
+---------
+
+- [Updated std's float parsing to use the Eisel-Lemire algorithm.][86761]
+  These improvements should in general provide faster string parsing of floats,
+  increases the range of accepted valid floating point values, and reduce
+  the produced code size for non-stripped artifacts.
+- [`string::Drain` now implements `AsRef<str>` and `AsRef<[u8]>`.][86858]
+- [`collections::{BinaryHeap, BTreeSet, HashSet, LinkedList, VecDeque}` now
+  implement `From<[T; N]>`.][84111]
+- [`collections::{BTreeMap, HashMap}` now implement `From<[(K, V); N]>`.][84111]
```

Comment:
Would it be worth mentioning how this allows for literal-like syntax, as [described in the PR](#84111)?